dashboard/agent.py

```diff
@@ -62,7 +62,9 @@ def __init__(self,
         self.object_store_name = object_store_name
         self.raylet_name = raylet_name
         self.node_id = os.environ["RAY_NODE_ID"]
-        assert self.node_id, "Empty node id (RAY_NODE_ID)."
```

Comment:
this wasn't checking anything -- an env variable won't be in os.environ if it's an empty string